src/controllers/action-queue/runtime.js

```diff
@@ -49,7 +49,7 @@ export const executeActionProgram = async (account: EdgeAccount, program: Action
       return { nextState: { ...state, effect: nextEffect } }
     } catch (error) {
       // Silently fail dryrun
-      showError(error)
+      console.error(error)
```

Comment:
This will redbox in development mode. It might be better to use `console.warn` or `console.log` instead.

Reply:
As long as warns show stack traces as well, then I agree (I think it does, but for some reason briefly thought it wouldn't).